<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterFaceClass</class>
 <widget class="QMainWindow" name="InterFaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InterFace</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="3">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item row="2" column="2">
     <widget class="QTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(File)</string>
    </property>
    <addaction name="menuSave"/>
    <addaction name="actionRead"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>管理(Manage）</string>
    </property>
    <addaction name="actionMpart"/>
    <addaction name="actionMinstance"/>
   </widget>
   <widget class="QMenu" name="menu_Create">
    <property name="title">
     <string>新建（Create）</string>
    </property>
    <addaction name="actionLeg_3"/>
    <addaction name="actionBody_2"/>
    <addaction name="actionHead"/>
   </widget>
   <widget class="QMenu" name="menu_Caculate">
    <property name="title">
     <string>计算(Caculate)</string>
    </property>
    <addaction name="actionCaculate"/>
   </widget>
   <addaction name="menu_Create"/>
   <addaction name="menu"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Caculate"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actioninit"/>
   <addaction name="actionYoZ"/>
   <addaction name="actionXoZ"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actioninit">
   <property name="icon">
    <iconset>
     <normaloff>:/TW/init.png</normaloff>:/TW/init.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
  </action>
  <action name="actionYoZ">
   <property name="icon">
    <iconset>
     <normaloff>:/TW/YOZ.png</normaloff>:/TW/YOZ.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>YOZ</string>
   </property>
  </action>
  <action name="actionLeg">
   <property name="text">
    <string>Leg</string>
   </property>
  </action>
  <action name="actionBody">
   <property name="text">
    <string>Body</string>
   </property>
  </action>
  <action name="actionCrossArm">
   <property name="text">
    <string>Head</string>
   </property>
  </action>
  <action name="actionSavePartLeg">
   <property name="text">
    <string>SavePartLeg</string>
   </property>
  </action>
  <action name="actionSavePartBody">
   <property name="text">
    <string>SavePartBody</string>
   </property>
  </action>
  <action name="actionSavePartHead">
   <property name="text">
    <string>SavePartHead</string>
   </property>
  </action>
  <action name="actionSaveTower">
   <property name="text">
    <string>SaveTower</string>
   </property>
  </action>
  <action name="menuSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionRead">
   <property name="text">
    <string>读取</string>
   </property>
  </action>
  <action name="actionXoZ">
   <property name="icon">
    <iconset>
     <normaloff>:/TW/XOZ.png</normaloff>:/TW/XOZ.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>XOZ</string>
   </property>
  </action>
  <action name="actionMpart">
   <property name="text">
    <string>塔部件</string>
   </property>
  </action>
  <action name="actionMinstance">
   <property name="text">
    <string>塔实例</string>
   </property>
  </action>
  <action name="actionLeg_2">
   <property name="text">
    <string>Leg</string>
   </property>
  </action>
  <action name="actionBody_2">
   <property name="text">
    <string>Body</string>
   </property>
  </action>
  <action name="actionHead">
   <property name="text">
    <string>Head</string>
   </property>
  </action>
  <action name="actionLeg_3">
   <property name="text">
    <string>Leg</string>
   </property>
  </action>
  <action name="actionCaculate">
   <property name="text">
    <string>计算</string>
   </property>
  </action>
  <action name="actionExist">
   <property name="text">
    <string>现有模型</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TW.qrc"/>
 </resources>
 <connections/>
</ui>
